main/acle.rst

```diff
@@ -217,7 +217,7 @@ Change history
    | ACLE Q3 2020       | 31/10/20           | Arm                | Version ACLE Q3    |
    |                    |                    |                    | 2020.              |
    +--------------------+--------------------+--------------------+--------------------+
-   | |release|          | |date-of-issue|    | Arm                | Version ACLE Q2    |
+   | 2021Q2             | 02/07/21           | Arm                | Version ACLE Q2    |
```

Comment:
We want to render the value in the field exactly as in the specs released in the PDF. Please set the date with the format `02 July 2021`, not `02/07/21`.